service_contracts.rst

```diff
@@ -234,7 +338,7 @@ MonitoringService
 =================
 
 The :ref:`MS` submits an up-to-date :term:`balance proof` on behalf of users who are offline when a channel is closed to prevent them from losing tokens. This could be done without a dedicated contract by calling `TokenNetwork.updateNonClosingBalanceProof <update-channel>` but then the MS would not be able to claim a reward for its work.
-To handle the rewards, the MonitoringService contract provides two functions. One for wrapping `updateNonClosingBalanceProof` and creating the reward and another one for claiming the reward after the settlement:
+To handle the rewards, the MonitoringService contract provides two functions. One for wrapping `updateNonClosingBalanceProof` and creating the reward and another one for claiming the reward after the settlement. The wrapper of `updateNonClosingBalanceProof` only works for service providers that are registered in ServiceRegistry:
```

Comment:
I would tend to say that calling `monitor` only works for registered services, instead. But that's two ways to say the same thing.

Reply:
Your way sounds better.  I'll change the whole sentence.  Instead of talking about "one for X and one for Y", I can use names of functions.

Reply:
Rewrote the paragraph so it talks about `claimReward()` and `monitor()` explicitly.